specificatie/genereervariant/openapi.json

```diff
@@ -314,6 +314,72 @@
               "type": "string"
             }
           },
+          {
+            "name": "postcode",
+            "in": "query",
+            "description": "Postcode behorende bij een nummeraanduiding.",
+            "required": false,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "pattern": "^[1-9]{1}[0-9]{3}[a-zA-Z]{2}$",
+              "type": "string",
+              "example": "6922KZ"
+            }
+          },
+          {
+            "name": "huisnummer",
+            "in": "query",
+            "description": "Huisnummer behorende bij een nummeraanduiding.",
+            "required": false,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "maximum": 99999,
+              "minimum": 1,
+              "type": "integer",
+              "example": 1
+            }
+          },
+          {
+            "name": "huisletter",
+            "in": "query",
+            "description": "Huisletter behorende bij een nummeraanduiding.",
+            "required": false,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "pattern": "^[a-zA-Z]{1}$",
+              "type": "string",
+              "example": "A"
+            }
+          },
+          {
+            "name": "huisnummertoevoeging",
+            "in": "query",
+            "description": "Huisnummertoevoeging behorende bij een nummeraanduiding.",
```

Comment:
kunnen we dit veranderen in: de huisnummertoevoeging van het adres

Reply:
hier zou ik uitleggen wat een huisnummertoevoeging is. Niet iedereen kent dit begrip.

In de API specificatie staat al:
> Een toevoeging aan een huisnummer of een combinatie van huisnummer en huisletter die door de gemeente aan een adresseerbaar object is gegeven.

Hier vind ik "Een toevoeging aan een huisnummer of een combinatie van huisnummer en huisletter" het relevante deel van de uitleg. Maar een voorbeeld zegt meer dan 1000 woorden, dus waarom niet via een voorbeeld:
"Bijvoorbeeld bij Belgiëlaan 1 A3 is 1 het huisnummer, A de huisletter en 3 de huisnummertoevoeging"

Reply:
Op sommige properties willen we geen description, want "dat is vanzelfsprekend" en op andere properties willen we wel een toelichting want "niet iedereen kent dit begrip". 
Wat is nu het criterium om wel of niet een description toe te voegen ? 
Ik snap nog steeds niet waarom we er last van hebben als er "vanzelfsprekende" descriptions in een OAS staan, maar ik kan er mee leven dat we die weghalen. Ik vind het lastig dat er geen éénduidig criterium is om te bepalen wanneer we dat doen.

Reply:
volgens mij is het criterium  heel duidelijk (de API moet eenvoudig bruikbaar zijn zonder vergaande kennis van het domein), alleen is het toepassen daarvan niet altijd eenvoudig.